modules/bmo-about-the-bare-metal-operator.adoc

```diff
@@ -0,0 +1,69 @@
+// This is included in the following assemblies:
+//
+// post_installation_configuration/bare-metal-configuration.adoc
+
+:_module-type: CONCEPT
+[id="bmo-about-the-bare-metal-operator_{context}"]
+= About the Bare Metal Operator
+
+Use the Bare Metal Operator (BMO) to provision, manage, and inspect bare-metal hosts in your cluster. 
+ 
+The BMO uses three resources to complete these tasks: 
+
+* `BareMetalHost`
+* `HostFirmwareSettings`
+* `FirmwareSchema`
+
+The BMO maintains an inventory of the physical hosts in the cluster by mapping each bare-metal host to an instance of the `BareMetalHost` custom resource definition. Each `BareMetalHost` resource features hardware, software, and firmware details. The BMO continually inspects the bare-metal hosts in the cluster to ensure each `BareMetalHost` resource accurately details the components of the corresponding host. 
+
+The BMO also uses the `HostFirmwareSettings` resource and the `FirmwareSchema` resource to detail firmware specifications for the bare-metal host. 
+
+The BMO interfaces with bare-metal hosts in the cluster by using the Ironic API service. The Ironic service uses the Baseboard Management Controller (BMC) on the host to interface with the machine. 
+
+Some common tasks you can complete by using the BMO include the following:
+
+* Provision bare-metal hosts to the cluster with a specific image
+* Format a host's disk contents before provisioning or after deprovisioning
+* Turn on or off a host
+* Change firmware settings
+* View the host's hardware details 
+
+DRAFT IMAGE: 
+
+image::BMO-Diagram-v2.png[DRAFT - PLACEHOLDER IMAGE]
+
+.BareMetalHost
+
+The `BareMetalHost` resource defines a physical host and its properties. When you provision a bare-metal host to the cluster, you must define a `BareMetalHost` resource for that host. For ongoing management of the host, you can inspect the information in the `BareMetalHost` or update this information. 
+
+The `BareMetalHost` resource features provisioning information such as the following:
+
+* URL of the operating system boot image
```

Comment:
Forgot to mention: there is another case, where this field won't be populated. Instead, `customDeploy.method` will be set to a name of a plugin that is used for installation. Starting with OCP 4.10, it is `install_coreos`. I'm not sure how to present this information without confusing everyone...

Reply:
Is this documented? I only see a reference to it in the BMH specs: https://docs.okd.io/4.11/rest_api/provisioning_apis/baremetalhost-metal3-io-v1alpha1.html#spec-customdeploy

I don't think we should include it at a high-level here unless the procedure is documented somewhere.

Reply:
Proposed a fix upstream https://github.com/metal3-io/baremetal-operator/pull/1205 but it's quite brief since it lacks openshift-specific details. I think OpenShift is the main consumer of this field. Without documenting it, we may start getting questions like "Why is my host in the provisioned state without any image details"